src/corner_kick/src/types/canvas.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * This file describes the datatypes used for the canvas
+ */
+
+/**
+ * Represents a parsed layer received from the websocket
+ */
+export interface ILayerMessage {
+    /**
+     * The layer number
+     */
+    layer: number;
+
+    /**
+     * Layer flags
+     */
+    flags: boolean[];
```

Comment:
What sorts of flags would we have here?

Reply:
Hoping to have a flag that tells the visualizer that the layer has not changed since last message, perhaps one that modifies the visibility of the layer. I'm trying to reserve this area for future development.

Reply:
Hmm, I think this is somewhere we should apply [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) if we're not already using these flags. We can always add these back later if needed without any hassle?

Reply:
Sure!

Reply:
Looks like these are still here?

Reply:
Removed flag field for both shapes and layer

Reply:
You are fast. Just pushed right now.